src/components/CookiesBanner/index.tsx

```diff
@@ -189,12 +190,18 @@ const CookiesBanner = (): ReactElement => {
     setShowIntercom(localSupportAndUpdates)
 
     if (!localAnalytics) {
-      removeCookies(GA_COOKIE_LIST)
+      removeCookies(GA_COOKIE_RE)
```

Comment:
Does this need a timeout like the Beamer cookies?

Reply:
No.